src/components/todoForm/TodoForm.tsx

```diff
@@ -36,13 +50,22 @@ const TodoForm: React.FC = () => {
         value={title}
       />
       <Spacer />
-      <Button
-        onClick={() => {
-          handleAddTodo(title);
-        }}
-      >
-        Add
-      </Button>
+      <Row>
+        <Button
+          onClick={() => {
+            handleAddTodo(title);
+          }}
```

Comment:
move to seperate func

Reply:
I have already moved it to handleAddTodo function